BLUESPAWN-client/src/mitigation/mitigations/MitigateV3338.cpp

```diff
@@ -22,30 +22,28 @@ namespace Mitigations {
 
 	bool MitigateV3338::MitigationIsEnforced(SecurityLevel level) {
 		LOG_INFO("Checking for presence of " << name);
-		
-		auto key = RegistryKey{ HKEY_LOCAL_MACHINE, L"System\\CurrentControlSet\\Services\\LanManServer\\Parameters", L"NullSessionPipes" };
-		if(key.ValueExists()){
-			LOG_VERBOSE(2, L"Located value for " + key.GetName());
-			auto values = key.Get<REG_MULTI_SZ_T>();
+
+		auto key = RegistryKey{ HKEY_LOCAL_MACHINE, L"System\\CurrentControlSet\\Services\\LanManServer\\Parameters" };
+		if(key.ValueExists(L"NullSessionPipes")){
+			auto values = *key.GetValue<std::vector<std::wstring>>(L"NullSessionPipes");
+			auto vGoodValues = std::vector<std::wstring>{};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'vGoodValues' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4967854)